### PR TITLE
Fix typos in src

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
Misspellings found by [codespell](https://github.com/codespell-project/codespell).

I have tried not to modify vendored/generated files that are clearly maintained elsewhere. Needs double-checking though.

### CODE REVIEW

File: src/bin/checkrad.in [hunk 1]
```diff
@@ -1293,7 +1293,7 @@ sub mikrotik_telnet {
   ($login, $password) = naspasswd($terminalserver, 1);
   return 2 if ($password eq "");
 
-  # MikroTik routeros doesnt tell us to which port the user is connected
+  # MikroTik router doesn't tell us to which port the user is connected
```
Comment (arr2036) [inline src/bin/checkrad.in:1296]: Incorrect amendment.  Should be RouterOS.

Comment (DimitriPapadopoulos) [reply src/bin/checkrad.in]: Fixed.

File: src/lib/unlang/try_priv.h [hunk 2]
```diff
@@ -19,7 +19,7 @@
  * $Id$
  *
  * @file unlang/try_priv.h
- * @brief Declarations for unlang trys
+ * @brief Declarations for unlang tries
```
Comment (arr2036) [inline src/lib/unlang/try_priv.h:22]: Incorrect.  This is for try blocks.  Could be changed to `try keywords`

**Comment by DimitriPapadopoulos** [comment]
I had to change back `acquire` → `aquire` to avoid this error message:
```
src/modules/rlm_cache/rlm_cache.c:105:36: error: too many arguments provided to function-like macro invocation
        return inst->driver->acquire(out, &inst->config, inst->driver_submodule->dl_inst->data, request);
                                          ^
src/freeradius-devel/io/atomic_queue.h:47:9: note: macro 'acquire' defined here
#define acquire(_var)         atomic_load_explicit(&_var, memory_order_acquire)
        ^
src/modules/rlm_cache/rlm_cache.c:105:9: warning: incompatible pointer to integer conversion returning 'const cache_acquire_t' (aka 'int (*const)(void **, const rlm_cache_config_t *, void *, struct request_s *)') from a function with result type 'int' [-Wint-conversion]
        return inst->driver->acquire(out, &inst->config, inst->driver_submodule->dl_inst->data, request);
               ^~~~~~~~~~~~~~~~~~~~~
src/modules/rlm_cache/rlm_cache.c:98:88: warning: unused parameter 'request' [-Wunused-parameter]
static int cache_acquire(rlm_cache_handle_t **out, rlm_cache_t const *inst, request_t *request)
```

File: src/lib/unlang/try_priv.h (hunk 2 above)
Comment (DimitriPapadopoulos) [reply src/lib/unlang/try_priv.h:22]: Or perhaps:
```suggestion
 * @brief Declaration for unlang try
```

**Pull request merged**